vignettes/detailed-canvas-export-info.Rmd

```diff
@@ -0,0 +1,105 @@
+---
+title: "Detailed Canvas Export Info"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Detailed Canvas Export Info}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(gradebook)
+```
+
+This vignette details the specific structure of exported Canvas files. For information on how to use Canvas grade data with Gradebook, view the "Working With Canvas Data" vignette instead. This vignette details in depth information about Canvas file exports for curious advanced users.
```

Comment:
We should do this across the vignettes but it's nice to include the link whenever we reference other vignettes... I think it's already done in the readMD if you need a reference